api/src/routers/first-meal.js

```diff
@@ -0,0 +1,10 @@
+import express from "express";
+import knex from "../database_client.js";
+
+
+const firstMeal = express.Router();
+firstMeal.get('/', async(req , res)=> {
+    const first_meal = await knex('meal').orderBy('id').limit(1)
```

Comment:
Leverage the query builder to make the code more readable, e.g. https://knexjs.org/guide/schema-builder.html#first

```suggestion
    const first_meal = await knex('meal').orderBy('id').first()
```